src/Api/ResetPasswordApiController.php

```diff
@@ -34,31 +33,33 @@ class ResetPasswordApiController extends BaseApiController
     /**
      * Get the response for a successful password reset.
      *
-     * @param  string  $response ID of language resource to send as response
+     * @param string  $response ID of language resource to send as response
      * @return Response
      */
     protected function sendResetResponse($response)
     {
-        return $this->json(new SuccessMessage(trans($response)));
+        return $this->json(new Message(trans($response)));
```

Comment:
Also, why **ErrorMessage** and **SuccessMessage** classes was replaced by **Message**?
The reason of using separate classes is that on response handler we can add 20 status code for instance of SuccessMessage and status code 400 for instance of ErrorMessage classes. Now success and failed response looks identical